apps/docs/content/components/VideoPlayer.mdx

```diff
@@ -102,6 +102,103 @@ import {VideoPlayer} from '@primer/react-brand'
 </VideoPlayer>
 ```
 
+## Custom play icon
+
+```jsx live
+<VideoPlayer
+  title="GitHub media player"
+  playIcon={() => <PlayIcon size={96} />}
+>
+  <VideoPlayer.Source
+    src="https://primer.github.io/brand/assets/example.mp4"
+    type="video/mp4"
+  />
+  <VideoPlayer.Track src="https://primer.github.io/brand/assets/example.vtt" />
+</VideoPlayer>
+```
+
+## Controlling programmatically
+
+The `VideoPlayer` component exposes a `useVideo` hook that can be used to control the video programmatically. To use the hook, the `VideoPlayer` component must be wrapped in a `VideoPlayer.Provider` component.
+
+Full documentation for the `useVideo` hook can be found [below](#usevideo-context).
+
+```tsx live
+() => {
+  const MyVideoPlayer = () => {
+    const {isPlaying, togglePlaying, seek} = useVideo()
+
+    return (
+      <Stack direction="vertical">
+        <VideoPlayer
+          title="GitHub media player"
+          showPlayPauseButton={false}
+          showSeekControl={false}
+          showCCButton={false}
+          showMuteButton={false}
+          showVolumeControl={false}
+          showFullScreenButton={false}
+        >
+          <VideoPlayer.Source
+            src="https://primer.github.io/brand/assets/example.mp4"
+            type="video/mp4"
+          />
+          <VideoPlayer.Track src="https://primer.github.io/brand/assets/example.vtt" />
+        </VideoPlayer>
+        <Stack direction="horizontal">
+          <Button onClick={() => togglePlaying()}>
+            {isPlaying ? 'Pause' : 'Play'}
+          </Button>
+          <Button onClick={() => seek(0)}>Go to start</Button>
+          <Button onClick={() => seek((t) => t + 5)}>Skip 5 seconds</Button>
+        </Stack>
+      </Stack>
+    )
+  }
+
+  return (
+    <VideoPlayer.Provider>
+      <MyVideoPlayer />
+    </VideoPlayer.Provider>
+  )
+}
+```
+
+<h2>useVideo Context</h2>
```

Comment:
Can we down level this to h3 and rename for clarity. We tend to prefer markdown headings unless you need to add additional attributes in there. Just a stylistic point...

```suggestion
### useVideo hook
```

Reply:
Not sure how I managed to start using HTML over markdown there, good catch, thanks